miniapps/subduction/3D/Subduction3D.jl

```diff
@@ -172,31 +173,29 @@ function main3D(li, origin, phases_GMG, igg; nx=16, ny=16, nz=16, figdir="figs3D
         # advect particles in memory
         move_particles!(particles, xvi, particle_args)
         # check if we need to inject particles
-        inject_particles_phase!(particles, pPhases, (pT, ), (T_buffer,), xvi)
+        inject_particles_phase!(particles, pPhases, (), (), xvi)
         # update phase ratios
         update_phase_ratios!(phase_ratios, particles, xci, xvi, pPhases)
 
         @show it += 1
         t        += dt
 
         # Data I/O and plotting ---------------------
-        if it == 1 || rem(it, 5) == 0
-            checkpointing(figdir, stokes, thermal.T, η, t)
+        if it == 1 || rem(it, 20) == 0
+            # checkpointing(figdir, stokes, thermal.T, η, t)
 
             if do_vtk
                 velocity2vertex!(Vx_v, Vy_v, Vz_v, @velocity(stokes)...)
                 data_v = (;
-                    T   = Array(thermal.T),
-                    τxy = Array(stokes.τ.xy),
-                    εxy = Array(stokes.ε.xy),
+                    T   = zeros(ni.+1...),
+                    phase_vertex = [argmax(p) for p in Array(phase_ratios.center)]
```

Comment:
shouldn't this be phase_ratios.vertex?
```suggestion
                    phase_vertex = [argmax(p) for p in Array(phase_ratios.vertex)]
```

Reply:
yes